lib/chitter.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps")
+    result.map { |peep| peep['message'] }
+  end
+
+  def self.create(peep:, timestamp:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
You're using this same piece of code in more than one place in this file. How might you get rid of this duplication?